app/services/exports/funding_schedule_data.rb

```diff
@@ -76,10 +76,12 @@ def sanitize_row(row)
       row.map { |value| sanitize(value) }
     end
 
-    def sanitize(value)
-      return value unless value.is_a?(String)
+    def academic_cycle_year
+      Time.zone.now.strftime("%m").to_i > 7 ? AcademicCycle.current.start_year.to_i : AcademicCycle.current.end_year.to_i
```

Comment:
I don't think we can assume this as we'll likely be showing historic years in the future. We need to get these from the payment schedule. The years are annoyingly on the amounts so we'll need to get the min and max from those. Possibly put that as a method on PaymentSchedule?